psydac/feec/global_projectors.py

```diff
@@ -110,7 +110,7 @@ def __init__(self, Hcurl, n_quads=None):
 
```

Comment:
the default None value for n_quads yields an error on L78 (same for the Hdiv projector)

Reply:
Yes, `dim` should be read from the space, not from the quadrature rule.

Reply:
Fixed in commit fcfed12